unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/NotificationComponent/ChatNotificationMessageComponentView.cs

```diff
@@ -190,6 +191,22 @@ public void SetIsPrivate(bool isPrivate)
         ForceUIRefresh();
     }
 
+    public void SetIsFriendRequest(bool isFriendRequest)
```

Comment:
im not sure if this is the correct approach, perhaps we should have a new notification component for friend requests, what do you think?

Reply:
Yeah, it was something that I was thinking and I didn't finally do it because both notification components were going to be very similar, but yeah... seeing this `SetIsFriendRequest`, I think it is a bit confussing and it worth it to create a new component. I will do it 👍

Reply:
Done!! I have created a new component `FriendRequestNotificationComponentView` for friend request notifications.